wpilibj/src/main/java/edu/wpi/first/wpilibj/Watchdog.java

```diff
@@ -87,25 +87,25 @@ public int hashCode() {
    * @return The time in seconds since the watchdog was last fed.
    */
   public double getTime() {
-    return Timer.getFPGATimestamp() - m_startTime;
+    return Timer.getFPGATimestamp() - m_startTimeSeconds;
   }
 
   /**
    * Sets the watchdog's timeout.
    *
-   * @param timeout The watchdog's timeout in seconds with microsecond resolution.
+   * @param timeoutSeconds The watchdog's timeout in seconds with microsecond resolution.
    */
-  public void setTimeout(double timeout) {
-    m_startTime = Timer.getFPGATimestamp();
+  public void setTimeout(double timeoutSeconds) {
+    m_startTimeSeconds = Timer.getFPGATimestamp();
```

Comment:
This doesn't make sense either.

Reply:
Why not? Timer.getFPGATimestamp() returns seconds.

Reply:
What's wrong with it?  Timer.getFPGATimestamp() returns time in seconds.

Reply:
I think it's the same discussion as above